src/templates/parseScriptTemplates.ts

```diff
@@ -193,3 +214,46 @@ export function parseScriptTemplates(rawResources: object, rawTemplates: object[
     }
     return templates;
 }
+
```

Comment:
Filed this issue to refactor as discussed https://github.com/Microsoft/vscode-azurefunctions/issues/505